website/project/model.py

```diff
@@ -3282,7 +3282,8 @@ def _validate_authorizer(self, user):
 
     def _on_reject(self, user, token):
         parent_registration = Node.find_one(Q('embargo', 'eq', self))
-        parent_registration.registered_from.add_log(
+        registered_from = parent_registration.registered_from
+        registered_from.add_log(
```

Comment:
I think this is fine how it was -- it doesn't seem necessary to assign `parent_registration.registered_from` to a variable since it's only used in one place. And it saves a line of code! :sunglasses: